src/Search/Search.js

```diff
@@ -67,31 +68,28 @@ function WikipediaSearch() {
           return response.json();
         })
         .then((data) => {
-          setPage(data.parse.sections)
+          setPage(data.parse.sections);
         })
         .catch((error) => {
           console.error('There was a problem with the fetch operation:', error);
         });
-    
+    }
   }, [initialResults]);
 
 // This useEffect will run when the page state is updated to find the controversies
   useEffect(() => {
```

Comment:
This useEffect is pretty complicated - can `words` and `findMatchingSections` be defined outside of the use effect (for readability?) I can picture some of this other functionality could be abstracted into their own methods as well